app/assets/stylesheets/layout.scss

```diff
@@ -0,0 +1,83 @@
+#outer-container,
+#topnav-container,
+#search-navbar-container,
+#sul-footer-container {
+  border: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.header-logo > a,
+h3.panel-title a {
+  border-bottom: none;
+}
+
+#topnav, #search-navbar,
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line